rs/sns/governance/src/governance.rs

```diff
@@ -5262,7 +5287,7 @@ impl Governance {
                 GovernanceError::new_with_message(ErrorType::PreconditionFailed, msg),
             );
 
-            return;
+            return self;
```

Comment:
The point of borrowing is kind of that we don't need to return the object being referenced.

IMHO worth exploring a bit more how to get rid of all these.